.gitignore

```diff
@@ -12,7 +12,16 @@ debian/kodi-pvr-iptvsimple.substvars
 debian/kodi-pvr-iptvsimple/
 debian/tmp/
 obj-x86_64-linux-gnu/
+CMakeCache.txt
```

Comment:
I believe most people, including me, build from within the tree. This is also true for IDEs.
Are there any issues with this commit?